<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Application</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item rowspan="2" row="0" column="0" >
    <widget class="QLabel" name="appIcon" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>64</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../data/data.qrc" >:/data/package.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="appName" >
     <property name="text" >
      <string>&lt;b>amsn&lt;/b></string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="appSummary" >
     <property name="text" >
      <string>A free and open source MSN Messenger clone</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../data/data.qrc" />
 </resources>
 <connections/>
</ui>
